CITi-UFPE/CITiMarco
# Feature/login screen

## Whats I did
- Made login home screen

## How to test 
1. yarn start
1. Check if the screen looks nice and you can use the log in with google functionality

Resolves #10
